sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log azure-core-amqp
+
+## Version 1.0.0-beta.9 (2020-01-02)
+- Add links on batch send operation in eventhubs
+- Update qpid-proton-j-extensions dependency
+- Shorten tracing span names
+
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-amqp/README.md)
+
+- Initial release. Please see the README and wiki for information on the new design.
```

Comment:
This part isn't true, there have been other releases of this library, this should be changed unless the plan is to switch this to GA before releasing the next round of updates.